views/NatureTrustWaterPropertiesInstrumentProfileProvisional.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS erddap."NatureTrustWaterPropertiesInstrumentProfileResearch";
```

Comment:
Looks like either wrong file name or wrong table name

Reply:
it was a the wrong query, I must have made an issue while copying and pasting those different ones